cfgov/cfgov/settings/base.py

```diff
@@ -274,3 +328,26 @@
     AWS_S3_CALLING_FORMAT = 'boto.s3.connection.OrdinaryCallingFormat'
 
     MEDIA_URL = os.environ.get('AWS_S3_URL') + '/f/'
+
+# Govdelivery
+
+GOVDELIVERY_USER = os.environ.get('GOVDELIVERY_USER')
+GOVDELIVERY_PASSWORD = os.environ.get('GOVDELIVERY_PASSWORD')
+GOVDELIVERY_ACCOUNT_CODE = os.environ.get('GOVDELIVERY_ACCOUNT_CODE')
+
+# LOAD OPTIONAL APPS
+# code from https://gist.github.com/msabramo/945406
+
+for app in OPTIONAL_APPS:
+    if app.get("condition", True):
+	try:
+	    __import__(app["import"])
+	except ImportError:
+	    pass
+	else:
+	    for name in app.get("apps",()):
+		if name not in INSTALLED_APPS:
```

Comment:
@kave  I'm not sure what you mean--  the "if" on 342 is always going to be true, unless the app in question defines a "condition" that is false.

It then tries to import whatever is in the "import" value in the dictionary.  If there is an ImportError, skip to the next app. `else` (which, in a try block, fires if there are no exceptions), apply the settings to INSTALLED_APPS, etc